inspirehep/modules/editor/api.py

```diff
@@ -287,3 +288,10 @@ def upload_files():
         full_url = fs.getsyspath(filename)
 
     return jsonify({'path': full_url})
+
+
+@blueprint_api.route('/linked_references', methods=['POST'])
+def get_linked_refs():
+    references = request.json
+    matched_refs = match_references(references)
```

Comment:
this throws when `source` is missing in any item of `raw_refs`. I think it'd be much better if you modify `match_references` so that it skips those ones and `links` the ones with `source`, rather than throwing an exception.